code/iomethods.py

```diff
@@ -518,5 +544,132 @@ def decode_time_from_string(timeString):
     print 'Error decoding time string: string does not match a predefined time format'
     return 0
     # **********************************************************************************************************************
+def write_np_array_to_ncdf(lon, lat, t, fileName, dirName):
+    '''
+        Purpose:: Convert a numPy array to netCDF
+
+        Inputs:: lon: a string representing the time units found in the file metadata
+                 lat: an integer representing the time interval found in the file's metadata
+                 t: A 3 dimensional numPy array holding temperatures in Kelvin.
+                    The first dimension is time and the second and third dimensions are longitude/latitude
+                 fileName: The name of the file to be written to
+                 dirName: Path to the directory to be written to
+        Returns:: None. It writes to a netCDF file with extension .nc
+    '''
+
+    ncdf = netCDF4.Dataset(dirName + '/' + fileName + '.nc', "w", format="NETCDF4")
+    ncdf.Conventions = "COARDS"  # Set global attributes
+    ncdf.calendar = "standard"
+    ncdf.comments = "File"
+    ncdf.model = "geos/das"
+    ncdf.center = "gsfc"
+
+    time = ncdf.createDimension("time", None)  # Create and set dimensions
+    longitude = ncdf.createDimension("longitude", size=9896)
```

Comment:
again, to make this more generic, these sizes should be detected according to the dims of lat,lon variables that were passed.